### DISCUSSION TITLE
Discussion of Feb 2022 repo survey results

### DISCUSSION THREAD

**Discussion by danmoseley** [dotnet/runtime#67767] [General]
Results at https://github.com/dotnet/runtime/issues/67766. 

This discussion is a good place to share any thoughts or suggestions you have for how we can make this repo better.

**Comment by danmoseley** [comment]
I thought it might be interesting to plot our "contributor count" in this repo over time. The first plot below is number of distinct community contributors that either created an issue or a PR in this repo in each 60 day period. (Of course, one can be a major contributor without doing either of those things, but it's a measure.). The second plot is the same, but for just PR's. What we see here is that our community size (by this measure) has been relatively stable for several years. It would be interesting to do a similar plot, but for _amount_ of contribution (by some measure, such as number of community PR's merged, or total LOC changed) to see whether we see a similar shape.

![image](https://user-images.githubusercontent.com/6385855/163233838-a72080e0-0cb0-4c81-9f80-5a3741605982.png)

**Reply by PathogenDavid** [reply]
That's pretty neat to see. Would it be easy to add the months overlayed on that? I'd be interested to see if there's any trends in that regard. (Like college students having time off school.)

**Reply by danmoseley** [reply]
Here. A little different - I must have used some slightly different bucket size above. The raw numbers on the y axis are probably not interesting, because they're very sensitive to bucket size. If I choose a 1-year bucket, for example, there's 2,000 unique issue openers.
![image](https://user-images.githubusercontent.com/6385855/163482170-7a8d0fb6-3f39-46f7-9141-9f9eeee960d7.png)

I don't see a seasonal signal, I think. Again though, didn't look at amount of contribution (yet)

**Comment by Symbai** [comment]
Why are the results only published for the runtime repo? The last poll results has shown the feedback across all repos which were available.

**Reply by danmoseley** [reply]
@Symbai this time around we had so many repos in the survey, and of different kinds -- varying in the type of community, their size, age, focus -- that comparing across them is less useful than comparing to past results. Any comparison would inevitably involve editorial, and the survey organizers don't want to be in the position of evaluating other repos. For this reason I'd encourage you to ask the maintainers of particular repos about their results and what changes they saw over time. You could ask on the original issue posting the poll in that repo.

I see Pyright have posted [this ](https://github.com/microsoft/pyright/issues/3246) for example. I understand that @captainsafia will be posting info about aspnetcore results some time next week.

**Reply by Symbai** [reply]
I'm not speaking about comparing repo against repo but showing the results for each repo individually. Just like you did here for runtime but then also for the other repo's.

The problem is that some repo owners do not react to this question. Especially the repo's who had very bad results when it was public posted but now it no longer is and I'm curios to see if its just my own experience that hasn't changed or if other people feel the same. But thanks for the answer.

**Reply by danmoseley** [reply]
Yeah. Ultimately you have to ask. It's not right for me to share their results.

**Reply by danmoseley** [reply]
Update: ASP.NET [posted theirs](https://github.com/dotnet/aspnetcore/issues/41242)
